src/components/app.tsx

```diff
@@ -194,6 +213,12 @@ export class App extends React.PureComponent<IProps, IState> {
     );
   }
 
+  private renderAuthError = () => {
+    return (
+      <AuthError />
+    );
+  }
```

Comment:
Since this function is just returning `<AuthError />` we probably don't need the `renderAuthError` render function.  We can just stick `<AuthError />` in where `renderAuthError` is being called and remove the `renderAuthError` function.